pkg/controllers/management/restrictedadminrbac/cluster_test.go

```diff
@@ -1,7 +1,4 @@
 // mocks created with the following commands
-//
-//	mockgen --build_flags=--mod=mod -package restrictedadminrbac -destination ./mocks_test.go github.com/rancher/rancher/pkg/generated/controllers/management.cattle.io/v3 ClusterRoleTemplateBindingController,ClusterRoleTemplateBindingCache,ClusterCache,GlobalRoleBindingCache
-//
 // mockgen --build_flags=--mod=mod -package restrictedadminrbac -destination ./mockIndexer_test.go k8s.io/client-go/tools/cache Indexer
```

Comment:
Do we have any plan to mock indexers as well? Just curious

Reply:
If you are using a wrangler cache the calls to the indexer are wrapped by `AddIndexer` and `GetByIndex`  and are mocked [here](https://github.com/rancher/wrangler/blob/master/pkg/generic/fake/cache.go#L39-L79). These tests also use the Norman caches which is why the indexer has to be mocked directly.